src/encoded/types/file.py

```diff
@@ -571,7 +571,8 @@ def _update(self, properties, sheets=None):
             "parent of": "derived from",
             "supercedes": "is superceded by",
             "is superceded by": "supercedes",
-            "paired with": "paired with"
+            "paired with": "paired with",
+            "grouped with": "grouped with"
```

Comment:
Did you add inserts with these relationships?  Also did you test to see if the relationships were made reciprocally when only one was provided in an excel spreadsheet loaded by submit4DN - locally?